lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {
+  NxThreatIndicatorLegendProps,
+  nxThreatIndicatorLegendPropTypes,
+  nxThreatPropTypes,
+  ThreatProps
+} from './types';
+import { NxH3, NxThreatIndicator } from '../..';
+import './NxThreatIndicatorLegend.scss';
+
+const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const ThreatIndicator = ({ threatType }: ThreatProps) => {
+  return (
+    <div>
+      <NxThreatIndicator threatLevelCategory={threatType} />
+      <span>{capitalize(threatType)}</span>
+    </div>
+  );
+};
+
+ThreatIndicator.propTypes = nxThreatPropTypes;
```

Comment:
Internal components don't need propTypes

Reply:
> Internal components don't need propTypes

Removed `propTypes` for internal component: 016cef08072f4685c8b748af55edb80afede751b